pages/emailSender.tsx

```diff
@@ -340,12 +351,30 @@ const EmailSender: NextPage = () => {
             <StyledButton
               color="info"
               variant="contained"
-              onClick={() => sendEmails()}
+              onClick={() => handleClickOpen()}
               width="medium"
               disabled={finalMessages.length === 0}
             >
               Send!
             </StyledButton>
+            <Dialog
+              open={open}
+              onClose={handleClose}
+              aria-labelledby="alert-dialog-title"
+              aria-describedby="alert-dialog-description"
```

Comment:
We don't have a DialogContent or DialogContentText, so no need for this `aria-describedby` label